modules/installation-osp-custom-subnet.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
+
+[id="installation-osp-custom-subnet_{context}"]
+= Custom subnets in {rh-openstack} deployments
+
+Optionally, you can deploy a cluster on a {rh-openstack-first} subnet of your choice. The subnet's GUID is passed as the value of `platform.openstack.machinesSubnet` in the `install-config.yaml` file.
+
+This subnet is used as the cluster's primary subnet; nodes and ports are created on it.
+
+Before you run the {product-title} installer with a custom subnet, verify that:
+
+* The target network and subnet are available.
+
+* DHCP is enabled on the target subnet.
+
+* You can provide credentials to the installer that have permission to create ports on the target network.
+
+* If your network configuration requires a router, it is created in {rh-openstack}. Some configurations rely on routers for floating IP address translation.
+
+* Your network configuration doesn't rely on a provider network. Provider networks are not supported.
+
+[NOTE]
+====
+By default, the API and Ingress VIPs take .5 and .7 respectively from your network's CIDR block. To override these default values,
```

Comment:
Can you rewrite minus "respectively"? ISG suggests avoiding this word. Also, "Put a 0 in front of the decimal separator when the decimal value is less than 1" (p. 155).

> By default, the API VIP takes 0.5 and the Ingress VIP takes 0.7 from your network's CIDR block.

Reply:
@bobfuru How about x.x.x.5 and x.x.x.7, instead? I think just going with 0.5 and 0.7 would be confusing in the context of IP addresses.

Reply:
Ah, I didn't look at this in the right context. Old habit, saw the decimal and defaulted to what I know. Could be fine as is, but having the x.x.x.5 does help, IMO.